package.json

```diff
@@ -13,9 +13,11 @@
     "start": "node dist/server.js"
   },
   "devDependencies": {
+    "@types/express": "^4.17.13",
     "@types/node": "^16.4.13",
     "@typescript-eslint/eslint-plugin": "^4.28.5",
     "@typescript-eslint/parser": "^4.28.5",
+    "dotenv": "^10.0.0",
```

Comment:
This should be a regular dependency. Since your code depends on it. A deployment service for example would also read an environment variable